corehq/apps/app_manager/static/app_manager/js/summary.js

```diff
@@ -0,0 +1,52 @@
+hqDefine("app_manager/js/summary.js", function() {
+    $(function() {
+        'use strict';
+
+        var v2 = COMMCAREHQ.toggleEnabled('APP_MANAGER_V2');
+        if (v2) {
+            hqLayout.utils.setIsAppbuilderResizing(true);
+        }
+
+        var summaryApp = window.angular.module('summaryApp', ['ngRoute', 'summaryModule']),
+            initial_page_data = hqImport("hqwebapp/js/initial_page_data.js").get,
+            url = hqImport('hqwebapp/js/urllib.js').reverse;
+        summaryApp.config(['$httpProvider', function($httpProvider) {
+            $httpProvider.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
+            $httpProvider.defaults.xsrfCookieName = 'csrftoken';
+            $httpProvider.defaults.xsrfHeaderName = 'X-CSRFToken';
+        }]);
+        summaryApp.config(["djangoRMIProvider", function(djangoRMIProvider) {
+            djangoRMIProvider.configure(initial_page_data('djng_current_rmi'));
+        }]);
+
+        summaryApp.constant('summaryConfig', {
+            staticRoot: initial_page_data('static_root'),
+            vellumTypes: initial_page_data('vellum_types'),
+            formNameMap: initial_page_data('form_name_map'),
+            appLangs: initial_page_data('app_langs'),
+            caseDownloadURL: url("download_case_summary"),
+            formDownloadURL: url("download_form_summary"),
+            appDownloadURL: url("download_app_summary"),
+            appSettingsURL: url("release_manager"),
+            appHomeURL: url("view_app"),
+            appName: initial_page_data("app_name"),
+        });
+
+        summaryApp.config(['$routeProvider', function($routeProvider) {
+            $routeProvider.
+                when('/forms', {
+                    templateUrl: url("ng_template", "form_summary_view" + (v2 ? "_v2" : "")),
+                    controller: 'FormController',
+                }).
+                when('/cases', {
+                    templateUrl: url("ng_template", "case_summary_view" + (v2 ? "_v2" : "")),
+                    controller: 'CaseController',
+                }).
+                otherwise({
+                    redirectTo: '/forms',
+                });
+        }]);
+
+        angular.bootstrap(document.getElementById("summary-app"), ["summaryApp"]);
```

Comment:
Explicitly initialize the angular app here rather than using the `ng-app` directive because `summaryApp` is now inside a document ready handler (because it depends on initial_page_data and urllib, both of which can't be guaranteed to be ready until the dom is ready), so it isn't going to be ready at the point where the `ng-app` directive would normally get called.